Dockerfile

```diff
@@ -1,7 +0,0 @@
-FROM python:2.7
```

Comment:
Why are we deleting the docker file?

Reply:
Ah sorry. I just saw that none of the other repos had it and I wasn't sure why it was there. I will add it back. Do we need to add a Dockerfile to the other repos?

Reply:
Eventually yes. We want to have a docker image for each of the libraries that one can pull down and use. We are adding them one by one.

Reply:
Understood. Thank you for clarifying.